app/assets/stylesheets/components/_button.sass

```diff
@@ -1,17 +1,22 @@
 $Button-color-background: Theme-color(button, primary)
 $Button-color-text: Theme-color(button, text)
-$Button-height: 50px
+$Button-fontSize: Theme-fontSize(text, base)
+$Button-fontWeight: Theme-fontWeight(regular)
+$Button-height: 40px
 $Button-spacing: $Theme-spacing-base
+$Button-weight: 200px
```

Comment:
You should be the one, not me, making sure that every changes you agree to do are actually done.